tools/SourceKit/docs/Protocol.md

```diff
@@ -40,8 +40,8 @@ provided.
 
 | Request Name | Request Key | Description |
 | -------------:|:------------|:------------|
-| `codecomplete` | `codecomplete` |  Given a file will open a code-completion session which can be filtered upon using `codecomplete.update`. Each session must be closed using `codecomplete.close`.|
-| `open` | `codecomplete.open` | Open a code-completion session for the given input file and offset, and return the initial list of completions. |
+| `codecomplete` | `codecomplete` | Open a code-completion session for the given input file and offset, and return the initial list of completions. |
```

Comment:
So, to clarify, `codecomplete` would be defined as "Returns completions given an input file and offeset". Does that mean (for my own understanding) it opens a session internally and closes it?